### PR TITLE
Update for document translation

### PR DESCRIPTION
**Pull request by kristapratico**
Trying to get my docs published to github.io

### CODE REVIEW

**Review by iscai-msft — APPROVED** [review]

File: _data/releases/latest/python-packages.csv [hunk 1]
```diff
@@ -241,7 +241,7 @@
 "","","","","Windows 10 IoT Core Services","","NA","NA","","","","","","true","",""
 "","","","","Windows Virtual Desktop","","NA","NA","","","","","","true","",""
 "","","","","Xamarin","","NA","NA","","","","","","true","",""
-"azure-ai-translation-document","","1.0.0b1","azure-ai-translation-document","","NA","NA","NA","","false","","","","","",""
+"azure-ai-translation-document","","1.0.0b1","Document Translation","Translator","translation","NA","NA","","false","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/python-packages.csv:244]: I'm not sure who the owner of the other language packages are but we should try and update these in a consistent way for all the packages.

Comment (weshaggard) [reply _data/releases/latest/python-packages.csv:244]: Also I should say that this should automatically update if the package name in the release item matched. See https://dev.azure.com/azure-sdk/Release/_workitems/edit/7567/ where the package name didn't match.

Comment (kristapratico) [reply _data/releases/latest/python-packages.csv:244]: My bad, there was a decision to change the package name the day before the release so it didn't cross my mind to update it here as well. I'll do an update for .NET as well (JS/Java did not release).

Comment (kristapratico) [reply _data/releases/latest/python-packages.csv:244]: https://github.com/Azure/azure-sdk/pull/2694

Comment (weshaggard) [reply _data/releases/latest/python-packages.csv:244]: Thanks. Please update the release items as well, including the .NET one also https://dev.azure.com/azure-sdk/Release/_workitems/edit/7565/

Comment (kristapratico) [reply _data/releases/latest/python-packages.csv:244]: I don't seem to have permissions to edit the package name for either work item. Do you know if I need to create a new one?

Comment (weshaggard) [reply _data/releases/latest/python-packages.csv:244]: To update set the state back to "New" and change the package name and the other fields to match what you added here. Then save and then you can change the state back to "In Release"

Comment (kristapratico) [reply _data/releases/latest/python-packages.csv:244]: Thank you so much! Updated both.

**Pull request merged**